UI/window-basic-main-outputs.cpp

```diff
@@ -870,6 +870,7 @@ void SimpleOutput::SetupVodTrack(obs_service_t *service)
 		clear_archive_encoder(streamOutput, SIMPLE_ARCHIVE_NAME);
 }
 
+#define SRT_PROTOCOL "srt"
```

Comment:
Can this be moved to the top of the file along with the other protocol definitions?

https://github.com/obsproject/obs-studio/blob/92d1e5a8ccace361bf5908d4c29062e44bd99600/UI/window-basic-main-outputs.cpp#L19-L20

Reply:
done